src/main/scala/strawman/collection/mutable/Set.scala

```diff
@@ -69,9 +69,7 @@ trait Set[A]
   }
 }
 
-object Set {
-  def apply[A](xs: A*): Set[A] = ???
-}
+object Set extends IterableFactory.Delegate[Set](HashSet)
```

Comment:
There is at least one place where we have to arbitrarily decide which collection implementation to return: the `Map` operations inherited from `Iterable`, like `map[B](f: ((K, V)) => B): Iterable[B]`. Let’s just return a concrete implementation that works fine in most cases…